edge-auth/handlers/query.go

```diff
@@ -30,7 +30,10 @@ func MakeQueryHandler(config *Config, protected []string, restrictedPrefix []str
 	customersURL := os.Getenv("customers_url")
 
 	customers := sdk.NewCustomers(customersPath, customersURL)
-	customers.Fetch()
+	if err := customers.Fetch(); err != nil {
+		log.Fatalf("Unable to fetch customers from configured source")
```

Comment:
Not sure if this should be fatal. What if it timed out?

Reply:
then the next time they try it might not? changed the log line and written a 500 internal server error instead.